rpa-mapper/src/test/resources/DefenceResponse/defence_response_organisation_rpa_case_alreadyPaid.json

```diff
@@ -0,0 +1,27 @@
+{
+  "caseNumber": "000CM001",
+  "issueDate": "26-Apr-2018",
+  "defenceResponse": "I have paid what I believe I owe",
+  "defendants": [
+    {
+      "type": "Organisation",
+      "name": "John Rambo",
+      "businessName": null,
+      "companiesHouseNumber": null,
+      "contactPerson": "Steven Seagal",
```

Comment:
can we stick to made up names please? 
just in case anyone gets antsy about it

Reply:
Sure.